guacamole/src/main/java/org/glyptodon/guacamole/net/basic/extension/Extension.java

```diff
@@ -142,10 +154,10 @@
 
     /**
      * Returns a new map of all resources corresponding to the map of resource
-     * paths provided. Each resource will be associated with the mimetype 
+     * paths provided. Each resource will be associated with the mimetype
```

Comment:
Please remove unnecessary whitespace changes, though I wish these weren't here in the first place... =/